docs/guide/protocols/absolutesolvation.rst

```diff
@@ -1,4 +1,94 @@
+while the Lennard-Jones interactions are decoupled, meaning the intermolecular interactions turned off, keeping the intramolecular Lennard-Jones interactions.
+
+The lambda schedule
+~~~~~~~~~~~~~~~~~~~
+
+Molecular interactions are turned off during an alchemical path using a discrete set of lambda windows. The electrostatic interactions are turned off first, followed by the decoupling of the van-der-Waals interactions. A soft-core potential is applied to the Lennard-Jones potential to avoid instablilites in intermediate lambda windows. 
+
+.. _Simulation details:
+
+Simulation details
+~~~~~~~~~~~~~~~~~~
+
+The protocol applies a LangevinMiddleIntegrator which uses Langevin dynamics, with the LFMiddle discretization [1]_.
+Before running the production MD simulation in the NPT ensemble, the protocol performs a minimization of the system, followed by an equilibration in the NPT ensemble. A MonteCarloBarostat is used in the NPT ensemble to maintain constant pressure.
+
+Getting the free energy estimate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The free energy differences are obtained from simulation data using the MBAR estimator (multistate Bennett acceptance ratio estimator).
+TODO: Link to results page once done
+
+In addition to the MBAR estimates of the two legs of the thermodynamic cycle and the overall absolute solvation free energy difference, the protocol also returns some metrics to help assess convergence of the results. The forward and reverse analysis looks at the time convergence of the free energy estimates. The MABR overlap matrix checks how well lambda states overlap. Since the accuracy of the MBAR estimator depends on sufficient overlap between lambda states, this is a very important metric. 
+To assess the mixing of lambda states in the Hamiltonian replica exchange method, the results object returns the replica exchange transition matrix, which can be plotted as the replica exchange overlap matrix, as well as a time series of all replica states. (Todo: link to the results page in case examples of these plots are deposited there) 
+
+Simulation overview
+-------------------
+
+The ``ProtocolDAG`` of the :class:`.AbsoluteSolvationProtocol` contains both the units from the vacuum and from the solvent transformations. 
+This means that both legs of the thermodynamic cycle are constructured and run concurrently in the same ``ProtocolDAG``. This is different from the :class:`RelativeHybridTopologyProtocol` where the ``ProtocolDAG`` only runs a single leg of a thermodynamic cycle.
+If multiple repeats of the protocol are run, the ``ProtocolDAG`` contains multiple units of both vacuum and solvent transformations. 
+For each ``ProtocolUnit`` in the ``ProtocolDAG`` first, a non-alchemical equilibration is carried out (minimization, NVT and NPT equilibration), followed by an alchemical production (see :ref:`Simulation details`).
```

Comment:
It might be easier (for your sanity when dealing with the other protocol and also for readers) to here put this as a list.

So maybe you can do 

```
Simulation Steps
+++++++++++

Each Protocol simulation Unit (whether vacuum or solvent) carries out the following steps:

1. Parameterize the system using OpenMMForceFields (ref) and Open Force Field (ref).
2. Equilibrate the fully interacting system using a short MD simulation using the same approach as the :class:`.PlainMDProtocol` (in the solvent leg this will include rounds of NVT and NPT equilibration)
3. Create an alchemical system
4. Minimize the alchemical system
5. Equilibrate  and production simulate the alchemical system using the chosen multistate sampling method (under NPT conditions if solvent is present).
6. Analyze results for the transformation

Note: three different types of multistate sampling (i.e. replica swapping between lambda states) methods can be chosen; HREX, SAMS, and independent (no lambda swaps attempted). By default the HREX approach is selected, this can be altered using <insert the relevant setting name>.
```